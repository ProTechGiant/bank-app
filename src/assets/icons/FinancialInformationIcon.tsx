import * as React from "react";
import Svg, { Path } from "react-native-svg";

import { IconProps } from "@/assets/icons";

export function FinancialInformationIcon({ width = 24, height = 24, color = "#FF371E" }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M19 14V6c0-1.1-.9-2-2-2H3c-1.1 0-2 .9-2 2v8c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2zm-2 0H3V6h14v8zm-7-7c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3zm13 0v11c0 1.1-.9 2-2 2H4v-2h17V7h2z"
        fill={color}
      />
    </Svg>
  );
}
