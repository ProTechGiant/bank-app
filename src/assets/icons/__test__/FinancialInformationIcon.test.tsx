import { create } from "react-test-renderer";

import { FinancialInformationIcon } from "../FinancialInformationIcon";

describe("icon/FinancialInformationIcon", () => {
  it("renders a FinancialInformationIcon", () => {
    const result = create(<FinancialInformationIcon />);

    expect(result).toMatchSnapshot();
  });
});
