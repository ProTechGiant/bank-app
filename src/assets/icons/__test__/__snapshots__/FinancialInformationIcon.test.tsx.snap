// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/FinancialInformationIcon renders a FinancialInformationIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M19 14V6c0-1.1-.9-2-2-2H3c-1.1 0-2 .9-2 2v8c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2zm-2 0H3V6h14v8zm-7-7c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3zm13 0v11c0 1.1-.9 2-2 2H4v-2h17V7h2z"
      fill={
        {
          "payload": 4294915870,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
